helm/kafka/scripts/create-topic.sh

```diff
@@ -18,8 +18,7 @@ get_current_acl() {
     # form the output with the principals. Then use sed to strip out everything
```

Comment:
After comparing this diff to https://github.com/UKHomeOffice/callisto-localdev/commit/a4ec3c2444b28d81c1d621fce81f9070b1a41a1b#diff-c884395578b83214809e2fbf5b807ae20b65760f636ec7119a9801315b3f44a2
Lines 13&14 shouldn't be needed